src/artemis.cpp

```diff
@@ -138,6 +146,25 @@ Packages_t ProcessPackages(std::unique_ptr<ParameterInput> &pin) {
   parthenon::MetadataFlag MetadataOperatorSplit =
       parthenon::Metadata::AddUserFlag("OperatorSplit");
 
+  if (do_coagulation) {
```

Comment:
I feel that coagulation need not use the "anonymous" operator split framework.  Even coupling with `jaybenne` has moved away from this design.  I'd mimic the handling of IMC for dust coagulation... e.g.,

```
if (do_coagulation) status = Dust::OperatorSplitDust<GEOM>(pmesh, tm.time, tm.dt);
if (status != TaskListStatus::complete) return status;
```

maybe we should completely remove the `OperatorSplitTasks` machinery?

Reply:
done as you suggested.